### DISCUSSION TITLE
Multiple Inner Join Queries.

### DISCUSSION THREAD

**Discussion by RafaelCENG** [supabase/supabase#20182] [Questions] [answered]
Hello everyone I'm using 3 tables.
Table A has:
user_id, (foreign key that shows to table B)
information

Table B has:
id,
game1_id (foreign key that shows to table C)
game2_id  (foreign key that shows to table C)
etc..  (foreign key that shows to table C)

Table C has
game_id, some other fields

First, I'm using table A to get the users that I need. Working perfectly. Then I need an other inner join to get my Table C information for each game.
```
.from("TableA")
.select("tableB(name,user_id,game1_id)")
```
Using that game1_id how can I get all the information from table C?

**Comment by GaryAustin1** [comment, marked as answer]
https://postgrest.org/en/stable/references/api/resource_embedding.html?highlight=embedded#multiple-many-to-one
This section of the REST API (PostgREST) is very useful for these types of things.
